### ISSUE TITLE
Slow blockchain sync after upgrade - медленная синхронизация 

### ISSUE THREAD

**Issue by bitcoiners** [arterynetwork/artr#6]
After upgrading to version 1.3 we restarted blockchain synchronization. 
Synchronization was very fast till about 509,000 block (took about 3 hours from genesis), then slowed down to about 10 blocks/minute.

После обновления до версии 1.3 мы начали синхронизацию с новым genesis файлом,
до примерно 509,000 блока грузка шла быстро (3 часа от genesis), но потом резко замедлилась и сейчас качает примерно по 5-10 блоков в минуту.

**Comment by Ruby3434** [comment]
У меня такая же проблема

**Comment by QDeathNick** [comment]
Да, до 50% синхронизируемых блоков ещё более менее двигалось, после совсем медленно.
Я так понимаю это связано с увеличивающимся количеством обрабатываемых кошельков.

Но при синхронизации он почти ничего не качает, только свежие блоки, уже в конце синхронизации.

**Comment by QDeathNick** [comment]
Там 1.3.2 уже давно работает, но быстрее не стало.

**Comment by Ruby3434** [comment]
где 1.3.2? только офф 1.3.1 вижу. Да и генезиса нового нету, то быстрее не будет понятное дело

**Comment by QDeathNick** [comment]
Тут да, а в винде уже 1.3.2

**Comment by Ruby3434** [comment]
только генезис там старый

**Comment by Ruby3434** [comment]
да на винде там вообще синк в минус уходит. новые блоки появляются раньше чем подтягиваются старые. На винде пробовал недавно, то было 62 процента, уже 51..

**Comment by QDeathNick** [comment]
Это у вас железо не справляется. Я купил M.2 диск, синк стал успевать за блокчейном.
Найдите меня в телеграме https://t.me/QDeathNick

**Comment by g00dwin555** [comment]
Синхронизация очень медленная, даже М2 диск не помогает, загрузка диска 30%. за 12 часов, только 9% синхронизировалось. В чем проблема?

**Comment by QDeathNick** [comment]
Ну во первых проблема в том с какого момента вы начали синхронизацию. Так что может и 9% это ещё хорошо. Смотрите на скорость обработки блоков, а не на проценты. Ну и тут не вижу смысла продолжать, если интересно, то жду в телеграм.

**Comment by Meirbek90** [comment]
А как в кошелёк заходить то???

**Comment by QDeathNick** [comment]
Добавить ключ из сид-фразы в keyring и потом пользоваться кошельком. 
Читайте мануалы по cosmos sdk. https://docs.cosmos.network/master/run-node/keyring.html
